src/pudl/metadata/resources/eia860.py

```diff
@@ -708,6 +708,70 @@
         "sources": ["eia860"],
         "etl_group": "eia860",
     },
+    "_core_eia860__fgd_equipment": {
+        "description": (
+            "Information about FGD equipment at generation facilities, "
+            "from EIA-860 Schedule 6E. Note: This table has been cleaned, but not "
+            "harvested with other EIA 923 or 860 data. The same variables present in "
+            "this table may show up in other _core tables in other years. Once this "
+            "table has been harvested, it will be removed from the PUDL database."
+        ),
```

Comment:
I think we should spell out what FGD means in the table description at least. It's used in many other names but never explicitly expanded as far as I've seen.